practices/pairProgramming/readme.md

```diff
@@ -0,0 +1,33 @@
+# Pair Programming
+
+Pair programming is the concept of two team members working on a single story/feature. Often, this is two developers working back and forth (there are numerous patterns/strategies to use) to deliver business value. Some high value pairs could be frontend and backend specialists. Each can help drive their specialty areas as well as help train the other. Additionally, QA and Devs can pair to ensure that their features are delivered with appropriate tests in place.
+
+## The What and the Why
+
+### The What
+
+Pair programming is a part of XP or [Extreme Programming](https://www.agilealliance.org/glossary/xp/) and consists of two programmers sharing the same workstation while working on a problem. One programmer types out solutions, called the "driver", while the second programmer focuses more on the overall direction, called the "navigator". It is expected that the two programmers periodically switch roles. When more than two programmers are involved this is called a swarm.
```

Comment:
while often associated with and a core tenet of XP, pair programming isn't exclusive to any agile methodology (or to agile)

Reply:
Fair enough! Although it may be good to still link it to Extreme Programming since Concord does follow those practices correct?